t/lib/MakeMaker/Test/NoXS.pm

```diff
@@ -13,12 +13,13 @@ require XSLoader;
 delete $DynaLoader::{boot_DynaLoader};
 
 if ($^O eq 'MSWin32') {
+    # Load Win32. Backup GetCwd and SetChildShowWindow. Then clear the stash of all other entries.
     require Win32;
-    my $GetCwd = *{'Win32::GetCwd'}{CODE};
-    my $SetChildShowWindow = *{'Win32::SetChildShowWindow'}{CODE};
-    %{*main::Win32::{HASH}} = ();
-    *{'Win32::GetCwd'} = $GetCwd;
-    *{'Win32::SetChildShowWindow'} = $SetChildShowWindow;
+    my $GetCwd = Win32->can('GetCwd');
```

Comment:
How about `\&Win32::GetCwd`?

Reply:
Is this a stylistic comment or is there a difference?

Reply:
It's a direct reference to the function so closer to what the original code did; there's no reason to care about inheritance here and the function names are static so no need for `->can` really.

Reply:
I have removed the use of can.